cases/transactions.test.js

```diff
@@ -0,0 +1,251 @@
+            toml.TRANSFER_SERVER + `/transactions?asset_code=${enabledCurrency}&limit=1`, {
+                headers: {
+                    Authorization: `Bearer ${jwt}`
+                }
+            }
+        );
+
+        const json = await response.json();
+        expect(response.status).toEqual(200);
+        expect(json.error).not.toBeDefined();
+        expect(json.transactions.length).toBe(1);
+    });
+
+    it("return proper transactions with no_older_than param", async () => {
+        const currentDate = new Date();
+        await createTransaction(enabledCurrency, keyPair.publicKey(), toml, jwt, true, true);
+        await createTransaction(enabledCurrency, keyPair.publicKey(), toml, jwt, true, false);
+
+        const response = await fetch(
+            toml.TRANSFER_SERVER + `/transactions?asset_code=${enabledCurrency}&no_older_than=${currentDate.toISOString()}`, {
+                headers: {
+                    Authorization: `Bearer ${jwt}`
+                }
+            }
+        );
+
+        const json = await response.json();
+        expect(response.status).toEqual(200);
+        expect(json.error).not.toBeDefined();
+
+        json.transactions.forEach((transaction) => {
+            const transactionStartedTime = new Date(transaction.started_at).getTime();
+            expect(transactionStartedTime).toBeGreaterThanOrEqual(currentDate.getTime());
+        });
+    });
+
+    it("return only deposit transactions with kind=deposit param", async () => {
+        await createTransaction(enabledCurrency, keyPair.publicKey(), toml, jwt, true, true);
+        await createTransaction(enabledCurrency, keyPair.publicKey(), toml, jwt, true, false);
+
+        const response = await fetch(
+            toml.TRANSFER_SERVER + `/transactions?asset_code=${enabledCurrency}&kind=deposit`, {
+                headers: {
+                    Authorization: `Bearer ${jwt}`
+                }
+            }
+        );
+
+        const json = await response.json();
+        expect(response.status).toEqual(200);
+        expect(json.error).not.toBeDefined();
+
+        json.transactions.forEach((transaction) => {
+            expect(transaction.kind).toBe("deposit");
+        });
+    });
+
+    it("return only withdrawal transactions with kind=withdrawal param", async () => {
+        await createTransaction(enabledCurrency, keyPair.publicKey(), toml, jwt, true, true);
+        await createTransaction(enabledCurrency, keyPair.publicKey(), toml, jwt, true, false);
+
+        const response = await fetch(
+            toml.TRANSFER_SERVER + `/transactions?asset_code=${enabledCurrency}&kind=withdrawal`, {
+                headers: {
+                    Authorization: `Bearer ${jwt}`
+                }
+            }
+        );
+
+        const json = await response.json();
+        expect(response.status).toEqual(200);
+        expect(json.error).not.toBeDefined();
+
+        json.transactions.forEach((transaction) => {
```

Comment:
should expect that there is at least one tx, this would pass with an empty array